exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/parser/ObjectListener.java

```diff
@@ -52,80 +52,64 @@
  */
 public interface ObjectListener {
 
+  public enum FieldType {
```

Comment:
No need to declare it explicitly as public. Everything declared in the interface is public by default.